### PR TITLE
Add 'DISTINCT FROM' operators

### PR DESCRIPTION
**Pull request by danysantiago**
Add `IS DISTINCT FROM` and `IS NOT DISTINCT FROM` operators that where added in SQLite [3.39.0](https://www.sqlite.org/changes.html#version_3_39_0)

### CODE REVIEW

File: sql/sqlite/SQLiteParser.g4 [hunk 1]
```diff
@@ -270,6 +270,8 @@ expr
         | NOT_EQ2
         | IS_
         | IS_ NOT_
+        | IS_ DISTINCT_ FROM_
+        | IS_ NOT_ DISTINCT_ FROM_
```
Comment (KvanTTT) [inline sql/sqlite/SQLiteParser.g4:274]: Please simplify:

```suggestion
        | IS_ NOT_? DISTINCT_ FROM_
```

**Review by KvanTTT — CHANGES_REQUESTED** [review]
Please cover your changes by an example in [sqlite/examples](https://github.com/antlr/grammars-v4/tree/master/sql/sqlite/examples) (add a new or extend an existing).

File: sql/sqlite/SQLiteParser.g4 (hunk 1 above)
Comment (danysantiago) [reply sql/sqlite/SQLiteParser.g4]: Done - Thanks

**Comment by danysantiago** [comment]
> Please cover your changes by an example in [sqlite/examples](https://github.com/antlr/grammars-v4/tree/master/sql/sqlite/examples) (add a new or extend an existing).

Of course! - Done

**Pull request merged**